__tests__/plugin.test.ts

```diff
@@ -63,23 +76,14 @@ describe('Plugin', () => {
                 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
                 // @ts-ignore
                 expect(result.css).toMatchCss(`
-                    .font-sans.text-sm.leading-sm.capsize,
-                    .font-sans .text-sm.leading-sm.capsize,
-                    .font-sans .text-sm .leading-sm.capsize,
-                    .text-sm .font-sans.leading-sm.capsize,
-                    .text-sm .font-sans .leading-sm.capsize {
-                        padding: 0.05px 0;
-                    }
-
                     .font-sans.text-sm.leading-sm.capsize::before,
                     .font-sans .text-sm.leading-sm.capsize::before,
                     .font-sans .text-sm .leading-sm.capsize::before,
                     .text-sm .font-sans.leading-sm.capsize::before,
                     .text-sm .font-sans .leading-sm.capsize::before {
                         content: '';
-                        margin-top: -0.3542em;
-                        display: block;
-                        height: 0;
+                        margin-bottom: -0.3506em;
+                        display: table;
```

Comment:
Guessing the difference in negative margin values is because the 0.05px padding isn't a factor any more?